base/commands/Motion2D.hpp

```diff
@@ -13,9 +15,11 @@ namespace base
     {
         double translation; //! translation value in m/s
         double rotation;    //! rotation value in rad/s. Positive is counter-clockwise
-        double heading;     //! heading in rad. Positive is counter-clockwise
-        Motion2D():translation(0), rotation(0), heading(0){};
-        Motion2D(double translation, double rotation, double heading):translation(translation), rotation(rotation), heading(heading){};
+        base::Angle heading;     //! heading in rad. Positive is counter-clockwise
+        
+        Motion2D():translation(0), rotation(0), heading(base::Angle::fromRad(0)){};
+        Motion2D(double translation, double rotation):translation(translation), rotation(rotation), heading(base::Angle::fromRad(0)){};
```

Comment:
Better use the default constructor. 
I initializes with base::unset<double>. In this case 
one can distinguish between not used and Zero..